### PR TITLE
UIDATIMP-138: Add form for new match profile action

### PR DESCRIPTION
**Pull request by YevheniiMaltsev**
## Purpose

Implement a new form for match profiles creation according to requirements in the [story](https://issues.folio.org/browse/UIDATIMP-138).

## Screenshots

save_button_available_after_fields_changes

discard_unsaved_changes_confirmation_dialogue

empty_required_field

### CODE REVIEW

File: src/components/MatchProfilesForm/MatchProfilesForm.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import "@folio/stripes-components/lib/variables.css";
```
Comment (VictorSoroka1) [inline src/components/MatchProfilesForm/MatchProfilesForm.css]: I assume you do not need this file.

Comment (YevheniiMaltsev) [reply src/components/MatchProfilesForm/MatchProfilesForm.css]: @VictorSoroka1 Do you mean MatchProfilesForm.css at all or importing of "@folio/stripes-components/lib/variables.css"?

Comment (VictorSoroka1) [reply src/components/MatchProfilesForm/MatchProfilesForm.css]: Whole file. It is not used anywhere in your code)

Comment (YevheniiMaltsev) [reply src/components/MatchProfilesForm/MatchProfilesForm.css]: @VictorSoroka1 Oh, you are right. I deleted file. Thanks for the comment

File: src/components/MatchProfilesForm/MatchProfilesForm.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```
Comment (VictorSoroka1) [inline src/components/MatchProfilesForm/MatchProfilesForm.js]: Add a note to the Changelog file.

Comment (YevheniiMaltsev) [reply src/components/MatchProfilesForm/MatchProfilesForm.js]: @VictorSoroka1 Thanks! Updated changelog file

**Pull request merged**